### ISSUE TITLE
Several Head-ups Need to be Mentioned

### ISSUE THREAD

**Issue by HQYang1979** [mortii/anki-morphs#102]
Now that we have 0.10, I think the following warrants being mentioned.

1) Extra Fields
1.1) Selecting the following extra fields will add these fields into the card types, which will inevitably cause a full sync. This might not be an issue if the collection is small, but for a large collection like mine >320m, it simply will not be able to sync (though I have other way around to sync it).
1.2) Selecting the following extra fields will also populate the fields with words and numbers and sentences, again, it might not be an issue if the collection is small, but for a collection size around 250m, populating these fields will most likely enlarge the collection size to be more than 320m, and again, rendering the collection not be able to sync.
![image](https://github.com/mortii/anki-morphs/assets/74698160/38b84606-80c2-40eb-a501-0d7378253111)

2) Recalc will modify the whole collection
Recalc will modify the whole collection, re-organizing the whole collection to a new order, again, this might not be an issue if the collection is small, but it will be a problem when syncing, if the internet connection is not stable or not fast enough because Recalc is modifying the whole collection, which means almost all notes are modified, forcing Anki to sync every single notes to the server. Yes, every Recalc will make Anki to sync almost all notes, I am not talking about a full sync.
2.1) Anki desktop, it is still tolerable.
2.2) Ankimobile, the paid app surely excels, the syncing is superb.
2.3) Ankidroid will be the most problematic, especially the recent versions, it will freeze/terminate any collection sync if Ankidroid is going background, and it takes annoying time to sync every single notes (not full sync), and several tries to get it done because it is very easy for your mobile going dark then rendering the syncing fail.
In a nutshell, it is almost not usable with Ankidroid.

**Comment by mortii** [comment]
@HQYang1979 Very good points!

> a large collection like mine >320m

What do you mean with 320m? 320 MB? 320 million cards?

**Comment by aleksejrs** [comment]
I guess if I don't tell AM to sort my Incremental Reading notes, it won't add the fields to them, but those who do use Incremental Reading, might not want am-unknowns to be added to such cards.

**Comment by HQYang1979** [comment]
![image](https://github.com/mortii/anki-morphs/assets/74698160/10b8378e-a8e3-4c47-b63d-27faa20aba12)
This file will prompt a warning dialog from Anki that it must be reduced to <320m when a full sync is forced.

**Comment by mortii** [comment]
>This file will prompt a warning dialog from Anki that it must be reduced to <320m when a full sync is forced.

@HQYang1979 perfect, thank you :)

How many cards do you have by the way? and how long does recalc take for you?

**Comment by mortii** [comment]
> Incremental Reading

@aleksejrs extra fields are completely optional, sorting works just fine without them.

**Comment by HQYang1979** [comment]
> How many cards do you have by the way? and how long does recalc take for you?

total 672,552 cards. Around 5-10 minutes per Recalc.

**Comment by mortii** [comment]
@HQYang1979 It is this problem right? https://faqs.ankiweb.net/are-there-limits-on-file-sizes-on-ankiweb.html

**Comment by HQYang1979** [comment]
> @HQYang1979 It is this problem right? https://faqs.ankiweb.net/are-there-limits-on-file-sizes-on-ankiweb.html

Yes, actually the size has been increased to 320M.

**Comment by mortii** [comment]
@HQYang1979 is this ok?

Extra fields page:

> **Note**: The first time you select an extra field you have to do a full sync upload to AnkiWeb. If you have _a lot_ of cards (500K+), then syncing might become a problem. Read more about it on the [Anki FAQ](https://faqs.ankiweb.net/are-there-limits-on-file-sizes-on-ankiweb.html).

Recalc page:

> **Note**: Recalc can potentially reorganize all your cards, which can cause long sync times. The [Anki FAQ](https://faqs.ankiweb.net/can-i-sync-only-some-of-my-decks.html) has some tricks you can try if this poses a significant problem.

**Comment by HQYang1979** [comment]
Yes, thank you!

**Comment by mortii** [comment]
updated:
- [extra fields](https://mortii.github.io/anki-morphs/user_guide/setup/settings/extra-fields.html)
- [recalc](https://mortii.github.io/anki-morphs/user_guide/usage/recalc.html)

Thanks!

**Comment by aleksejrs** [comment]
> @aleksejrs extra fields are completely optional, sorting works just fine without them.

So what? I want them for some note types and I don't want them for other note types.

**Comment by aleksejrs** [comment]
@mortii
> @aleksejrs extra fields are completely optional, sorting works just fine without them.

That's not relevant; I want some all fields for some note types and only some for other note types.

**Comment by Vilhelm-Ian** [comment]
Not the appropriate place to continue the Convo aleks

**Comment by aleksejrs** [comment]
@Vilhelm-Ian mortii should see a notification and know where to continue better than I do.
